geozero/src/csv/csv_writer.rs

```diff
@@ -9,12 +9,12 @@ pub struct CsvWriter<'w, W: Write> {
     headers: Vec<String>,
     has_written_first_record: bool,
     current_row_props: Vec<String>,
-    wkt_writer: BufferingWktWriter,
+    wkt_writer: WktWriter<Vec<u8>>,
```

Comment:
It finally occurred to me that WktWriter could just as well own it's writer (perhaps inspired by https://github.com/georust/geozero/pull/157)